flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -132,9 +132,9 @@
 
 
 
-	private final Configuration config;
+	private Configuration config;
```

Comment:
I removed final from clientTimeout and config because I need to reassign them in [CliFrontend.java#L219](https://github.com/alkagin/flink/blob/bb9203923edeb02258455712db579d09dd3e7e62/flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java#L219), as far as I understand `--configDir` option must be available just in `run` mode. Is it correct? Otherwise I can read it in the main and instantiate correctly CliFrontend passing the configDir.